File: for-local-test/191.number-of-1-bits.java

class Solution {
    public int hammingWeight(int n) {
        int count = 0;
        while(n != 0){
            count++;
            n &= (n - 1);
        }
        return count;
    }
    public static void main(String[] args) {
        Solution solution = new Solution();
        int n = 11; // Example input
        int result = solution.hammingWeight(n);
        System.out.println("Number of 1 bits in " + n + " is: " + result);
    }
}

File: for-local-test/python/191.number-of-1-bits.py
class Solution:
    def hammingWeight(self, n: int) -> int:
        count = 0
        while n != 0:
            count += 1
            n &= (n - 1)
        return count
    
if __name__ == "__main__":
    solution = Solution()
    n = 11  # Example input
    result = solution.hammingWeight(n)
    print(f"Number of 1 bits in {n} is: {result}")